### ISSUE TITLE
Spotify Playlists not working since Spotify API Change 

### ISSUE THREAD

**Issue by thescottyq** [pimusicbox/pimusicbox#486]
Known issue with mopidy-spotify not being able to load the tracks within Spotify Playlists since Spotify updated their API

This will need to be fixed in Mopidy before being ported into pimusicbox, this thread is just to track this issue

**Comment by paradxum** [comment]
Can you please verify that this is the issue we are talking about? 
[https://github.com/mopidy/mopidy-spotify/issues/182](https://github.com/mopidy/mopidy-spotify/issues/182)
Does anyone know a work-around to be able to use Spotify at all right now?

**Comment by kingosticks** [comment]
> Can you please verify that this is the issue we are talking about?
Yes
> Does anyone know a work-around to be able to use Spotify at all right now?
Use the Mopify web client.

**Comment by thescottyq** [comment]
The Mopify Web Client on PiMusicbox does not load Spotify Playlists though. We are waiting on that fix to be rolled into the next build

**Comment by kingosticks** [comment]
It doesn't? Why not? Mopify loads playlist data from the Spotify Web API directly. This should work.

**Comment by rainerda** [comment]
> Use the Mopify web client.
I'm waaay too stupid to find it :-/

**Comment by thescottyq** [comment]
I can confirm that is does not work, even after a rebuild. We have ours set to use Mopify as the default client, we can see the playlist but when we click on it nothing loads

**Comment by jo3rg** [comment]
still open?

**Comment by AlbertMN** [comment]
Still having this issue,

**Comment by rainerda** [comment]
sure, it's a long work in progress:
https://github.com/mopidy/mopidy-spotify/pull/188
tldr; they have to switch to spotify web-api for that
don't bother moving to volumio, same problem there. use spotify-connect in the meantime.

**Comment by MrGJones** [comment]
I've had same problem with these music apps... found the spotify addon for
kodi works better!

**Comment by noah1510** [comment]
I can actually play playlists using spotify connect using a little trick.
First start the playlist on your computer and then select the speaker.
To play something different you first have to select your computer as output again start playing and then select musikbox again.
This workaroud works reliably on v0.7.0RC7
